src/bindings/js/node/include/pre_post_process_wrap.hpp

```diff
@@ -39,6 +39,7 @@ class PrePostProcessorWrap : public Napi::ObjectWrap<PrePostProcessorWrap> {
     Napi::Value set_input_tensor_shape(const Napi::CallbackInfo& info);
     Napi::Value set_input_tensor_layout(const Napi::CallbackInfo& info);
     Napi::Value set_input_model_layout(const Napi::CallbackInfo& info);
+    Napi::Value set_input_element_type(const Napi::CallbackInfo& info);
```

Comment:
Can you add here brief info about parameters of this method?
Something like this:
`/// @brief  
`/// @param info[0] index of the input`
`/// @param info[1] element_type`

Reply:
Added